diff --git a/drivers/media/dvb-frontends/stv6111.h b/drivers/media/dvb-frontends/stv6111.h
index 55b5ddc..066dd70 100644
--- a/drivers/media/dvb-frontends/stv6111.h
+++ b/drivers/media/dvb-frontends/stv6111.h
@@ -9,7 +9,7 @@ extern struct dvb_frontend *stv6111_attach(struct dvb_frontend *fe,
 #else
 
 static inline struct dvb_frontend *stv6111_attach(struct dvb_frontend *fe,
-				struct i2c_adapter *i2c, u8 adr);
+				struct i2c_adapter *i2c, u8 adr)
 {
 	pr_warn("%s: Driver disabled by Kconfig\n", __func__);
 	return NULL;
